gradle.properties

```diff
@@ -1,4 +1,4 @@
-projectVersion=4.0.0-SNAPSHOT
+projectVersion=9.0.0-SNAPSHOT
```

Comment:
I think this is not intentional, right? :)

Reply:
Whoopsie...  I was living in the future... 😊